states/state.py

```diff
@@ -5,21 +5,28 @@
 
 from config.config import TEXT
 
-T = TypeVar('T')
+T = TypeVar("T")
 
 
 class StateOptions(IntEnum):
+    """Gives Menu Option values"""
```

Comment:
`"""Base enum class for possible actions in game state`

Reply:
Changed to `Base enum class for possible actions in game stat`

Reply:
I sure hope it's `game state` and not `game stat`

Reply:
It's definitely `game state` now